## 通用参数
float64 maxSpeed
## 规划器参数
float64 vehicleRadius
float64 terrainVoxelSize
int8 useMap
float64 close_map_time
float64 adjacentRange
float64 pathRangeStep
float64 minSpeedRange
float64 minPathRange
int8 pathScaleBySpeed
float64 obstacleHeightThre
float64 dirWeight
float64 dirThre
float64 detourWei
float64 goalClearRange
int8 usePathScale
float64 pathScale
float64 minPathScale
float64 pathScaleStep
int8 use_fail_closemap